CHANGELOG

```diff
@@ -1,6 +1,29 @@
 devel
 -----
 
+* Added a new subquery optimization called `splice-subqueries`.
+
+	This optimisation splices the execution of subqueries inside the
+	execution of their surrounding query. For subqueries with a fairly
+	small number of returned rows this saves a significant amount of
```

Comment:
rows -> documents?

Reply:
Agreed. In that case, it's interchangeable, and more understandable.